samples/app/MeshAssimp.cpp

```diff
@@ -112,32 +115,73 @@ std::string shaderFromConfig(MaterialConfig config) {
     )SHADER";
 
     if (!config.unlit) {
+        if (config.normalUV == 0) {
+            shader += R"SHADER(
+                material.normal = texture(materialParams_normalMap, getUV0()).xyz * 2.0 - 1.0;
```

Comment:
Don't duplicate the whole fragment, all you need is to parameterize `getUV0()` (the 0/1 actually).